akka-persistence/src/main/scala/akka/persistence/fsm/PersistentFSM.scala

```diff
@@ -366,11 +366,8 @@ object PersistentFSM {
       copy(replies = replyValue :: replies)
     }
 
-    /**
-     * Modify state transition descriptor with new state data. The data will be
-     * set when transitioning to the new state.
-     */
-    @InternalApi
```

Comment:
wouldn't hurt to keep the internal marker

Reply:
Ok, why not.